templates/todo/projects/nodejs-mongo/.repo/bicep/README.md

```diff
@@ -46,7 +46,8 @@ The fastest way for you to get this application up and running on Azure is to us
 1. Run the following command to initialize the project, provision Azure resources, and deploy the application code.
 
 ```bash
-azd up --template todo-nodejs-mongo
+azd init --template todo-nodejs-mongo
+azd up
 ```
```

Comment:
Break this into two sections, 

1. Run the following command to initialize the project.  This command will clone the code to your current folder and prompt you for the following information:

(include the section below here)

```bash
azd init --template todo-nodejs-mongo
```

2. Run the following command to both provision the template's infrastructure to Azure and also deploy the applciation code to those newly provisioned resources.
```bash
azd up
```